client/src/components/TeamAlphaDetails/TeamAlphaDetails.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import DeveloperProfile from "../DeveloperProfile/DeveloperProfile";
+import "./TeamAlphaDetails.css";
+
+const teamAlphaData = [
+  {
+    name: "Saghar Hosseinardi",
+    role: "Full-Stack Developer",
+    githubUsername: "SagharHosseinmardi",
+    linkedinUsername: "Saghar Hosseinmardi",
+  },
+  {
+    name: "Andriana Saffo",
+    role: "Full-Stack Developer",
+    githubUsername: "AndrianaOS",
+    linkedinUsername: "Andriana Saffo",
+  },
+  {
+    name: "George Primentas",
+    role: "Front-End Developer",
+    githubUsername: "GeorgePrimentas",
+    linkedinUsername: "George Primentas",
+  },
+  {
+    name: "Gayle Thompson Igwebike",
+    role: "Full-Stack Developer",
+    githubUsername: "Gayle-Thompson-Igwebike",
+    linkedinUsername: "Gayle Thompson-Igwebike",
```

Comment:
the linkedin should be set to each developers profile: For eg, mine would be : www.linkedin.com/in/gayle-thompson-igwebike-624004228
currently the linkedin username shows that these pages dont exist.... Ask everyone for their link to make it easier for you.

Reply:
Hi Gayle thank you for letting me know, i fixed it and the link is working now.